crypto/src/signer.rs

```diff
@@ -32,14 +33,11 @@ impl EcdsaPub {
     }
 }
 
-pub type EcdsaPriv = CryptoBuf;
+pub type PrivKey = CryptoBuf;
```

Comment:
nit: I'm not convinced this needs to change names, `PrivKey` sounds a little too generic to me.

Reply:
Ahhh cool, sorry I didn't notice that connection. LGTM.